datafusion/sql/src/unparser/dialect.rs

```diff
@@ -15,19 +15,30 @@
 // specific language governing permissions and limitations
 // under the License.
 
+use regex::Regex;
+use sqlparser::keywords::ALL_KEYWORDS;
+
 /// Dialect is used to capture dialect specific syntax.
```

Comment:
```suggestion
/// `Dialect` to usse for Unparsing
///
/// The default dialect tries to avoid quoting identifiers unless necessary (e.g. `a` instead of `"a"`)
/// but this behavior can be overridden as needed
```

Reply:
Thanks. Look nice.